packages/flutter/lib/src/gestures/constants.dart

```diff
@@ -95,3 +94,12 @@ const double kMaxFlingVelocity = 8000.0; // Logical pixels / second
 /// tap in a jump-tap gesture.
 // TODO(ianh): Implement jump-tap gestures.
 const Duration kJumpTapTimeout = Duration(milliseconds: 500);
+
+/// Like [kTouchSlop], but for mouse pointers.
+const double kMouseHitSlop = 1.0; // Logical pixels;
```

Comment:
Should we be using the word "Mouse" in these?  I think something like "kPrecisePointerHitSlop" is more correct, since it wouldn't have to be a mouse (trackpads would have this set too, for instance). We use "precise" as terminology elsewhere (but we also use mouse some places too).

Reply:
Renamed to follow the precisePointer pattern